docs:  fix context information in guides (#2902)
keptn/lifecycle-toolkit: docs/docs/guides/otel.md

```

You can access the metrics from your browser at: `http://localhost:9999`

> # Advanced tracing configurations in Keptn: Linking traces

Reviewer: you can only have one h1 at the top of the file so this will need to be an h2 or h3

In Keptn you can connect multiple traces, for instance to connect deployments 
of the same application through different stages. 
To create connections between the traces of versions of your application, you can enrich the
`KeptnAppContext` resource with

> [Opentelemetry span links](https://opentelemetry.io/docs/concepts/signals/traces/#span-links).

Reviewer: ```suggestion
[OpenTelemetry span links](https://opentelemetry.io/docs/concepts/signals/traces/#span-links).
```

You can retrieve the span link from the JSON representation of the trace in Jaeger, where
it has the following structure:

```yaml
00-<trace-id>-<span-id>-01
```

> This span link needs to be added the `KeptnAppContext` resource, 

Reviewer: ```suggestion
This span link needs to be added to the `KeptnAppContext` resource, 
```

which ensures the connection

> between traces of different versions of the application.

Reviewer 2: ```suggestion
Use this value to populate the `spanLinks` field
of your `KeptnAppContext` resource
to connect traces of different versions of the application.
```
This makes it so that I find this info if I do a ctrl-f for `spanLinks`

```yaml
{% include "./assets/otel/keptn-app-context-span.yaml" %}
```

> **Note**
> Please be aware that the span link is just an example, and you need to retrieve the traceID and spanID
> of the application trace stored in the `KeptnAppVersion` resource you want to link.

> To store this new information in the traces, you need to increment the version

Reviewer: to end the previous note, you need a newline
```suggestion

To store this new information in the traces, you need to increment the version
```

```

You can access the metrics from your browser at: `http://localhost:9999`

# Advanced tracing configurations in Keptn: Linking traces

In Keptn you can connect multiple traces, for instance to connect deployments 

> of the same application trough different stages. 

Reviewer 2: ```suggestion
of the same application through different stages. 
```

To create connections between the traces of versions of your application, you can enrich the
`KeptnAppContext` resource with
[Opentelemetry span links](https://opentelemetry.io/docs/concepts/signals/traces/#span-links).
You can retrieve the span link from the JSON representation of the trace in Jaeger, where
it has the following structure:

```yaml
00-<trace-id>-<span-id>-01
```

> This span link needs to be added the the `KeptnAppContext`, which will ensure the connection

Reviewer 2: ```suggestion
This span link needs to be added the the `KeptnAppContext` resource, 
which ensures the connection
```

between traces of different versions of the application.

```yaml
apiVersion: lifecycle.keptn.sh/v1beta1
kind: KeptnAppContext
metadata:
  name: keptndemoapp
  namespace: keptndemo
spec:
  metadata:
    commit-id: "1234"
    author: "myUser"
  spanLinks:
    - "00-c088f5c586bab8649159ccc39a9862f7-f862289833f1fba3-01"
```

> **Note**
> Please be aware that the span link is just an example, and you need to retrieve the traceID and spanID
> of the application trace stored in the KeptnAppVersion resource you want to link.

> To store this new information in the traces, you need to bump the version 

Reviewer 2: ```suggestion
To store this new information in the traces, you need to increment the version 
```
